Formula/bombadillo.rb

```diff
@@ -0,0 +1,21 @@
+class Bombadillo < Formula
+  desc "Non-web browser, designed for a growing list of protocols"
+  homepage "http://bombadillo.colorfield.space"
+  url "https://tildegit.org/sloum/bombadillo/archive/2.2.0.tar.gz"
+  sha256 "037a5e70b02fcf34b7d01f710111576b66faf117bceb000b527bff13ce939f3c"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"bombadillo"
+    man1.install "bombadillo.1"
+    prefix.install_metafiles
+  end
+
+  test do
+    browser = IO.popen([bin/"bombadillo", "gopher://bombadillo.colorfield.space"])
```

Comment:
You're not really testing anything here

Reply:
Thanks for the feedback! There was a good test in the formula for `nnn` that was useful here.